src/ChildProcessUtilities.js

```diff
@@ -33,7 +36,8 @@ export default class ChildProcessUtilities {
 
   static execSync(command, opts) {
     const mergedOpts = objectAssign({
```

Comment:
Heh, [yeah](https://github.com/asini/asini/pull/15#issuecomment-248407293).  Thanks for pointing this out.  Very timely. 😁